doc/tcti-spi-ftdi.md

```diff
@@ -0,0 +1,60 @@
+# SPI TCTI FTDI
+
+The SPI TCTI FTDI can be used for communication with a SPI-based TPM over the FTDI MPSSE
+USB to SPI bridge. The FTDI module utilizes the `tcti-spi-helper` library for handling the
+PTP SPI protocol and the `libftdi-dev` library for handling the USB to SPI communication.
+
+Example of a FTDI MPSSE USB to SPI bridge is the product "USB 2.0 Hi-Speed to MPSSE
+Cable (SPI/I2C/JTAG master) with +3.3V digital level signals", part no: C232HM-DDHSL-0.
+Connect the cable to your TPM as specified in the table below:
+
+| C232HM-DDHSL-0 | Description |
```

Comment:
Could you write this as follows, so it's also readable with a text-editor ?
|   C232HM-DDHSL-0   | Description |
|         --         |     --      |
|   VCC, red, pin 1  |     VCC     |
|  SK, orange, pin 2 |    SCLK     |
|  DO, yellow, pin 3 |    MOSI     |
|  DI, green, pin 4  |    MISO     |
|  CS, brown, pin 5  |     CS      |
| GND, black, pin 10 |     GND     |

Reply:
Fixed.